pkg/kubelet/types/types.go

```diff
@@ -95,8 +95,8 @@ type Reservation struct {
 	Kubernetes v1.ResourceList
 }
 
-// A pod UID which has been translated/resolved to the representation known to kubelets.
+// ResolvedPodUID a pod UID which has been translated/resolved to the representation known to kubelets.
```

Comment:
Sorry, I wasn't clear in my original comment - we should also add the `is a` change for `ResolvedPodUID` and `MirrorPodUID` similar to `SortedContainerStatuses`.

Reply:
fixed